### PR TITLE
Add code to support a background queue

### PR DESCRIPTION
**Pull request by iHiD**
This adds a rake task which moves up to 10 jobs onto the end of the queue every second.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,8 +19,8 @@ gem 'aws-sdk-dynamodb', '~> 1.51'
 gem 'aws-sdk-s3'
 gem 'redis'
 gem 'mandate'
-gem 'exercism-config', '>= 0.76.0'
-# gem 'exercism-config', path: '../exercism_config'
+# gem 'exercism-config', '>= 0.76.0'
+gem 'exercism-config', path: '../exercism_config'
```
Comment (ErikSchierboom) [inline Gemfile:23]: This should be reverted before merging I think

File: lib/orchestrator/retrieve_next_job.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ def call
       locked_key = Exercism::ToolingJob.key_for_locked
 
       job_id = redis.lpop(queued_key)
-      return unless job_id
+      return unless job_id && job_id != ""
```
Comment (ErikSchierboom) [inline lib/orchestrator/retrieve_next_job.rb:10]: No `blank?` available here :(

File: lib/orchestrator/process_background_queue.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      loop do
+        break if should_exist
+
+        sleep(1)
+        execute
+      rescue StandardError => e
+        # If stuff goes wrong should we abort the machine
+        # or should we just give it a shot. Maybe have a counter?
+        puts e
```
Comment (ErikSchierboom) [inline lib/orchestrator/process_background_queue.rb:27]: Will we somehow be notified if something goes wrong?

**Review by ErikSchierboom — APPROVED** [review]

File: lib/orchestrator/retrieve_next_job.rb (hunk 2 above)
Comment (iHiD) [reply lib/orchestrator/retrieve_next_job.rb:10]: There's `empty?` actually. I'll change it to that :)

File: lib/orchestrator/process_background_queue.rb (hunk 3 above)
Comment (iHiD) [reply lib/orchestrator/process_background_queue.rb:27]: No :(

I'll add an issue for adding bugsnag to this.

**Pull request merged**